sample/src/main/java/com/alfresco/sample/MainViewModel.kt

```diff
@@ -210,12 +213,34 @@ class MainViewModel(private val context: Context) : ViewModel() {
                     facetFormat = "V2"
                 )
 
+                /*
+                *  val where = "(EXISTS(target/file) OR EXISTS(target/folder))"
+        val include = AlfrescoApi.csvQueryParam("path", "allowableOperations")
+        val orderBy = listOf("title ASC")
+                *
+                */
+
+                val where = "(EXISTS(target/file) OR EXISTS(target/folder))"
+                val include = AlfrescoApi.csvQueryParam("path", "allowableOperations")
+                val orderBy = listOf("title ASC")
+
+                val responseFavorite  = favoritesApi.listFavorites(
+                    AlfrescoApi.CURRENT_USER,
+                    0,
+                    25,
```

Comment:
Can we pass the static 25 value from a variable so that it can be changed at any time?

Reply:
It's just a sample class and add can add only static values here